docker/version_keys.py

```diff
@@ -0,0 +1,5 @@
+version_keys = {
```

Comment:
Can we use json file instead. I don't think we should be creating a separate python file for this

Reply:
I used a python file for 2 main reasons
1. Simple json file has the overhead of opening the JSON file and loading its contents into a Python object, and use it further (boils down to the same as using a python dict)
2. Secondly, if for unsupported versions, we might want to remove GPG_KEYS, working with python dicts would be much simpler for any automation script we add. 

LMK if these sound valid justifications for using a python dict as a new python file.

Reply:
I think we should use json if we are using a separate file. It's just a more standard way of maintaining data like this imo. Overhead would be minimal imo.

Reply:
This change has been made